depinject/errors.go

```diff
@@ -1,11 +1,40 @@
 package depinject
 
 import (
+	"fmt"
 	"reflect"
 
 	"github.com/pkg/errors"
 )
 
+type ErrMultipleImplicitInterfaceBindings struct {
+	error
+	Interface reflect.Type
+	Matches   []reflect.Type
+}
+
+func (err ErrMultipleImplicitInterfaceBindings) Error() string {
+	return fmt.Sprintf("Multiple implementations found for interface %v: %v", err.Interface, err.Matches)
+}
+
+type ErrExplicitBindingNotFound struct {
```

Comment:
not following the difference in these errors

Reply:
`ErrMultipleImplicitInterfaceBindings` multiple bindings of the same interface found without an explicit one.

`ErrExplicitBindingNotFound` the type mentioned in the explicit binding was not found.  Could probably use a better name?